### PR TITLE
Add AWS v11.1.2 to allow IC update

### PR DESCRIPTION
**Pull request by paurosello**
As ICC has 2 clusters with 11.1.0 and 2 clusters in 11.1.1, releasing version 11.1.2 for everyone would allow updating those clusters to the latest version.

11.1.0 and 11.1.1 will not be merged to master, will be kept in a branch

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- version: 11.1.1
+- version: 11.1.2
   state: wip
   date: 2020-03-17T12:00:00Z
```
Comment (sslavic) [inline aws.yaml]: While at it, change release date if you have better estimate for it

File: release-notes/aws/v11.1.2.md [hunk 2]
```diff
@@ -25,14 +25,15 @@
...
 
 - Removed CPU and memory limits to improve reliability.
 
-## coreDNS (GS v1.1.6)
+## coreDNS (GS v1.1.7)
 
 - Add Pod Disruption Budget.
 - Allow custom forward configuration destination and options.
 - Add `autopath` variable in the values file to make possible to configure or disable the plugin.
+- Disable autopath by default
```
Comment (sslavic) [inline release-notes/aws/v11.1.2.md]: This and previous entry one can be merged into one

File: app/changelog/net-exporter.yaml [hunk 3]
```diff
@@ -1,3 +1,8 @@
+- version: 1.7.0
```
Comment (sslavic) [inline app/changelog/net-exporter.yaml]: We tend to do this in separate PRs

File: app/changelog/net-exporter.yaml [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+  description: Ignore non-existing pods
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-03-20
```
Comment (sslavic) [inline app/changelog/net-exporter.yaml]: Wrong link, should be for 1.7.0

File: app/changelog/coredns.yaml [hunk 5]
```diff
@@ -1,3 +1,9 @@
+- version: 1.1.8
+  componentVersion: 1.6.5
+  description: Use cluster.kubernetes.clusterDomain instead of cluster.kubernetes.domain for custom DNS suffix.
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-20
```
Comment (sslavic) [inline app/changelog/coredns.yaml]:
```suggestion
    - https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20
```

File: release-notes/aws/v11.1.2.md [hunk 6]
```diff
@@ -25,14 +25,15 @@
...
 
-- Change name of the interfaces to manage for AWS CNI. See the [app changelog](https://github.com/giantswarm/kiam-app/blob/master/CHANGELOG.md#v121-2020-03-14) for details.
+- Change name of the interfaces to manage for AWS CNI from *calico* to *eni*. See the [app changelog](https://github.com/giantswarm/kiam-app/blob/master/CHANGELOG.md#v121-2020-03-14) for details.
 
 ## kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.4](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v104))
 
 - Removed CPU and memory limits to improve reliability.
 
-## coreDNS (GS v1.1.6)
+## coreDNS (GS v1.1.7)
```
Comment (sslavic) [inline release-notes/aws/v11.1.2.md]: ### coredns v1.6.5 ([Giant Swarm app v1.1.7](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-19))

File: release-notes/aws/v11.1.2.md [hunk 7]
```diff
@@ -25,14 +25,15 @@
...
 ## kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.4](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v104))
 
 - Removed CPU and memory limits to improve reliability.
 
-## coreDNS (GS v1.1.6)
+## coredns v1.6.5 ([Giant Swarm app v1.1.7](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-19))
 
 - Add Pod Disruption Budget.
 - Allow custom forward configuration destination and options.
 - Add `autopath` variable in the values file to make possible to configure or disable the plugin.
```
Comment (sslavic) [inline release-notes/aws/v11.1.2.md]:
```suggestion
- Make `autopath` plugin configurable, optional and disabled by default.
```

File: release-notes/aws/v11.1.2.md [hunk 8]
```diff
@@ -25,14 +25,14 @@
...
 
-- Change name of the interfaces to manage for AWS CNI. See the [app changelog](https://github.com/giantswarm/kiam-app/blob/master/CHANGELOG.md#v121-2020-03-14) for details.
+- Change name of the interfaces to manage for AWS CNI from *calico* to *eni*. See the [app changelog](https://github.com/giantswarm/kiam-app/blob/master/CHANGELOG.md#v121-2020-03-14) for details.
 
 ## kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.4](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v104))
 
 - Removed CPU and memory limits to improve reliability.
 
-## coreDNS (GS v1.1.6)
+## coredns v1.6.5 ([Giant Swarm app v1.1.7](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-19))
```
Comment (sslavic) [inline release-notes/aws/v11.1.2.md]:
```suggestion
## coredns v1.6.5 ([Giant Swarm app v1.1.8](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20))
```

**Comment by sslavic** [comment]
@paurosello link in https://github.com/giantswarm/releases/blob/master/README.md#aws has to be updated as well due to 11.1.1 -> 11.1.2 rename

File: app/changelog/coredns.yaml [hunk 9]
```diff
@@ -1,3 +1,9 @@
+- version: 1.1.8
+  componentVersion: 1.6.5
+  description: Use cluster.kubernetes.clusterDomain instead of cluster.kubernetes.domain for custom DNS suffix.
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v118-2020-03-20
```
Comment (sslavic) [inline app/changelog/coredns.yaml]:
```suggestion
```
Extracted this in separate PR https://github.com/giantswarm/releases/pull/185

File: app/changelog/net-exporter.yaml [hunk 10]
```diff
@@ -1,3 +1,8 @@
+- version: 1.7.0
+  description: Ignore non-existing pods
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#v170-2020-03-20
```
Comment (sslavic) [inline app/changelog/net-exporter.yaml]:
```suggestion
```
Extracted this into separate PR https://github.com/giantswarm/releases/pull/184

File: aws.yaml [hunk 11]
```diff
@@ -1,6 +1,6 @@
-- version: 11.1.1
+- version: 11.1.2
   state: wip
-  date: 2020-03-17T12:00:00Z
+  date: 2020-03-20T12:00:00Z
```
Comment (sslavic) [inline aws.yaml]:
```suggestion
  date: 2020-03-23T12:00:00Z
```

**Comment by paurosello** [comment]
Would we merge this now and modify later if needed?

**Comment by sslavic** [comment]
Release notes need more work for sure - e.g. inconsistent entries format for apps.

Still, both release definition and release notes are WIP, so sure, can be merged to master and continued later with follow up PRs.

**Comment by sslavic** [comment]
@paurosello btw a PO like @marians of firecracker or your team PO has to own finalizing release notes, test and announce release to all aws customers as part of publishing/activating the platform release.

**Comment by paurosello** [comment]
yes, @marians know and we will prepare an announcement, blog post and few other things

**Comment by marians** [comment]
Closing in favour of AWS 11.2.0
